### ISSUE TITLE
[Feature Request] Add Wireless Firmware & Network-Manager.

### ISSUE THREAD

**Issue by ghost** [FyshOS/fyshos#4]
Our wireless network story isn't great at the minute; In-tow with what the default (ie: non 'non-free' / firmware) Debian iso provides. We should include the 'non-free' repo by default and this (probably incomplete) list of driver packages:

> firmware-linux-nonfree
> firmware-iwlwifi 
> firmware-atheros
> firmware-realtek

Also, we should include the  `network-manager` package; Which is now the standard way in Linux to connect to this stack. Said package also appears to ship 'nmtui' a nice little terminal-bound front-end to manage it -- which is great to have, until we have something user-facing on our end.

**Comment by andydotxyz** [comment]
Sounds good. Ideally the network module in FyneDesk would have basic support for choosing / connecting to WiFi.

**Comment by ghost** [comment]
Yeah, nmtui would def be a stopgap solution -- but good to have, I think, in any case.

**Comment by ghost** [comment]
I mentioned this in the Discord a few days ago, but for convenience / future-reference:

It looks like in Debian 12 (Bookworm) there is now a dedicated `non-free-firmware` repo; Most if-not all of the listed packages above will be found there in 12 and onwards.

**Comment by andydotxyz** [comment]
Debian 12 will be an enabler for various Fyne tooling things too, excited that they have started the freeze process!
